src/asset-management/EulerV2Manager.sol

```diff
@@ -10,63 +10,51 @@ import { Address } from "@openzeppelin/utils/Address.sol";
 
 import { IAssetManagedPair } from "src/interfaces/IAssetManagedPair.sol";
 import { IAssetManager, IERC20 } from "src/interfaces/IAssetManager.sol";
-import { IPoolAddressesProvider } from "src/interfaces/aave/IPoolAddressesProvider.sol";
-import { IPool } from "src/interfaces/aave/IPool.sol";
-import { IAaveProtocolDataProvider } from "src/interfaces/aave/IAaveProtocolDataProvider.sol";
-import { IRewardsController } from "src/interfaces/aave/IRewardsController.sol";
+import { IDistributor } from "src/interfaces/merkl/IDistributor.sol";
+import { IERC4626 } from "lib/forge-std/src/interfaces/IERC4626.sol";
 
-contract AaveManager is IAssetManager, Owned(msg.sender), ReentrancyGuard {
+contract EulerV2Manager is IAssetManager, Owned(msg.sender), ReentrancyGuard {
     using FixedPointMathLib for uint256;
     using SafeCast for uint256;
 
-    event Pool(IPool newPool);
-    event DataProvider(IAaveProtocolDataProvider newDataProvider);
-    event RewardsController(IRewardsController newRewardsController);
     event Guardian(address newGuardian);
     event WindDownMode(bool windDown);
+    event VaultForAsset(IERC20 asset, IERC4626 vault);
     event Thresholds(uint128 newLowerThreshold, uint128 newUpperThreshold);
     event Investment(IAssetManagedPair pair, IERC20 token, uint256 shares);
     event Divestment(IAssetManagedPair pair, IERC20 token, uint256 shares);
 
-    /// @dev tracks how many aToken each pair+token owns
+    error OutstandingSharesForVault();
+
+    /// @dev Mapping from an ERC20 token to an Euler V2 vault.
+    /// This implies that for a given asset, there can only be one vault.
+    /// If the admin of the manager wishes to specify a different vault for an asset, they would have to manually ensure that all pairs have
+    /// divested, otherwise the pairs might not be able to retrieve their assets.
```

Comment:
These lines should wrap at the same width (80 or 100 col)

Reply:
okay fixed. Also made forge fmt format the comments as well. Previously it was excluded. However, line_length remains at the default at 120 for now